MongoDbGenericRepository/ReadOnlyMongoRepository.cs

```diff
@@ -15,6 +15,10 @@ namespace MongoDbGenericRepository
     /// </summary>
     public abstract class ReadOnlyMongoRepository : ReadOnlyMongoRepository<Guid>, IReadOnlyMongoRepository
     {
+        public ReadOnlyMongoRepository()
```

Comment:
? not sure how it would work a connection string?

Reply:
used by InitRepository